app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1> Quotes Search Results </h1>
```

Comment:
According to the user story, this page should also display the total count of results found as well as the top 10 results (or however many there are if fewer than 10).